### PR TITLE
Add <meta> tag for referrer same-origin also to new default tpl

### PR DESCRIPTION
**Pull request by kramred**
somehow this did not make it into the new template:
`<meta name="referrer" content="same-origin">`
it is in the vintage tpl (see also #692) but not in the new default tpl

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,6 +9,11 @@ pagecache
 .buildpath
 .project
 
+# macOS General
+.DS_Store
+.AppleDouble
+.LSOverride
+
```
Comment (ArthurHoaro) [inline .gitignore]: Those are related to your environment and not this project. Please add them to your [global `.gitignore` file](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) instead (the same thing applies to the Eclipse exclude btw, it should be removed).

**Comment by ArthurHoaro** [comment]
Good catch!

**Comment by kramred** [comment]
I removed my environment specific additions to the .gitignore file to make the commit cleaner

**Comment by ArthurHoaro** [comment]
Thanks!

File: .gitignore (hunk 1 above)
Comment (kramred) [reply .gitignore]: Thanks, didn't know about the global `.gitignore` file. I found the following links useful as well:
[A collection of useful .gitignore templates](https://github.com/github/gitignore)
[Web interface and more to create/merge .gitignore templates](https://www.gitignore.io/)

**Pull request merged**